attestation-agent/kbs_protocol/src/client/rcar_client.rs

```diff
@@ -179,11 +182,23 @@ impl KbsClient<Box<dyn EvidenceProvider>> {
         Ok(())
     }
 
-    async fn generate_evidence(&self, runtime_data: String) -> Result<String> {
+    async fn generate_evidence(
+        &self,
+        tee: Tee,
+        runtime_data: String,
+        nonce: String,
+    ) -> Result<String> {
+        debug!("Challenge nonce: {nonce}");
```

Comment:
not sure we need this

Reply:
If possible, we can have this remain for trouble shooting purpose?